packages/edit-post/src/components/header/header-toolbar/style.scss

```diff
@@ -39,11 +39,91 @@
 
 	// The Toolbar component adds different styles to buttons, so we reset them
 	// here to the original button styles
-	.edit-post-header-toolbar__left > .components-button.has-icon,
-	.edit-post-header-toolbar__left > .components-dropdown > .components-button.has-icon {
+	.edit-post-header-document-toolbar__left > .components-button.has-icon:first-child {
+		margin-right: 10px;
```

Comment:
Can be $grid-unit-10 probably.

Reply:
Spacing wise, generally 16px ($grid-unit-20) and 8px ($grid-unit-10) are used. In this screenshot:

<img width="1079" alt="Screenshot 2023-03-29 at 15 43 18" src="https://user-images.githubusercontent.com/1204802/228557874-ecde46c1-270c-4959-88f3-5479aa59f4cf.png">

16px between blue plus button and row of controls, and 8px between every icon and separator.